arduino/libraries/WiFi/src/WiFi.h

```diff
@@ -122,7 +123,7 @@ class WiFiClass
   bool _staticIp;
   tcpip_adapter_ip_info_t _ipInfo;
   uint32_t _dnsServers[2];
-
+  char _defaultHostname[HOSTNAME_MAX_LENGHT];
```

Comment:
Should this be called `_hostname` since it stores the "active" hostname?

Reply:
Should we also add a `+ 1` to the array size to account for the `\0` terminator?

Reply:
Yes i'm agree with the name and yes i'll add the +1;